pymess/version.py

```diff
@@ -1,4 +1,4 @@
-VERSION = (0, 12, 0)
+VERSION = (0, 13, 0)
```

Comment:
This is typically done during version bumping, but i guess you will do it right after merging, so OK.